### ISSUE TITLE
Feature Request: Add ability for index-pbench (server side) to accept command line operational parameters

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#325]
Feature Request: Add ability for `index-pbench` server side to accept command line operational parameters.

The use case for this is integration with [Browbeat](https://github.com/openstack/browbeat).

We want to enable browbeat to direct how the pbench-server code should behave, add the ability to endow pbench results with metadata about the browbeat environment in which it was run, and have browbeat direct pbench to which elasticsearch index it should use.

To that end, we believe browbeat would invoke `index-pbench` as follows (basically, replacing how the `pbench-index` cron job [invokes it](https://github.com/distributed-system-analysis/pbench/blob/master/server/pbench/bin/pbench-index#L136)):

```
index-pbench --config <browbeat>/pbench-index.cfg --metadata="{ \"browbeat_uuid\": \"abcd1234...\", ... }" <tarball>
```

@atheurer, @jtaleric, @ndokos

**Comment by ashishkamra** [comment]
@ndokos provide updates

**Comment by ndokos** [comment]
This was implemented by commit 3a270cab2aad9b8f3e32450356453e8e4959cd5b with a date of Oct 20, 2016. It was merged on March 20, 2017, as part of PR #525 and was released as part of v0.43 on June 22, 2017.
